app/views/autosign/_form.html.erb

```diff
@@ -1,9 +1,13 @@
 <%= form_tag smart_proxy_autosign_index_path(@proxy) do |f| %>
-  <div class="clearfix">
-    <%= label_tag(_("Name")) %>
-    <div class="input">
-      <%= text_field_tag(:id) %>
+  <%= render :layout => 'common/modal', :locals => { :id => 'autosignModal',
+                                                     :title => _("New Autosign Entry"),
+                                                     :buttons => [ modal_close(_('Cancel')),
+                                                                   submit_tag(_('Save'), :class => "btn btn-success") ]} do %>
+    <div class="clearfix">
+      <%= label_tag(_("Name")) %>
```

Comment:
Or, more precisely, no - since f here was fictive, as autosign is not an object.  But I made the layout more similar to our other forms.